### ISSUE TITLE
Standard disaggregations missing on QA

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#1163]
This system message seems to be appearing on every result form on QA:

![image](https://user-images.githubusercontent.com/33670923/52661246-78601880-2eb6-11e9-9eba-f29beb3928a6.png)

I've never seen this message before, and my understanding is that every single program and every single indicator is supposed to have the SADD standard disaggregations hard-wired in. 

Some additional weirdness -- this is program 544 that has custom disaggregations set up in additional to standard. Note the system message with two types of disaggregation below, that seem to be lacking the styling Paul introduced with Huckleberry -- blue links instead of gray boxes, etc.

![image](https://user-images.githubusercontent.com/33670923/52663279-1d7cf000-2ebb-11e9-893d-c8d4336de142.png)

**Comment by axoplasm** [comment]
I'm going to split the styling issue onto a separate ticket (#1165), it's pretty straightfoward.

The other thing though…that one is really weird. Looks like @camgnostic might be able to shed some light on that:

```bash
[paul@Aphelocoma TolaActivity]$ git blame templates/indicators/results/result_form_disaggregation_fields.html | ack organizations
93ad20bfb (Cameron Mcfee 2018-12-20 01:29:06 -0800  36)         <p>{% trans "Standard disaggregations are entered by the administrator for the entire organizations.  If you are not seeing any here, please contact your system administrator." %}</p>
```

**Comment by axoplasm** [comment]
Testing this on QA, I notice an errant character:

![image](https://user-images.githubusercontent.com/169060/52742000-9d728b00-2f8b-11e9-9e26-954fd582ffcc.png)

**Comment by axoplasm** [comment]
…looks like Cameron fixed this in 1942771

**Comment by RickieT** [comment]
**Deployment Verification**

- [x] Confirmed “Ready for Deploy” status
- [x] No longer see errant "/" as reported 
- [x] Reported error no longe occurs as originally pictured/written
